### PR TITLE
commit 77a4a86 from https://github.com/andreasscherbaum/ansible-lxc-ssh

### PR DESCRIPTION
**Pull request by stefangweichinger**
belongs to #1712

### CODE REVIEW

File: ansible/plugins/connection_plugins/lxc_ssh.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
 # Copyright 2016 Pierre Chifflier <pollux@wzdftpd.net>
-# SPDX-License-Identifier: GPL-3.0-or-later
```
Comment (drybjed) [inline ansible/plugins/connection_plugins/lxc_ssh.py]: SPDX identifier is required by DebOps.

Comment (stefangweichinger) [reply ansible/plugins/connection_plugins/lxc_ssh.py]: re-added in 455757438

**Comment by drybjed** [comment]
@stefangweichinger Looks like you didn't pass the changes through `pycodestyle`, you can do it locally by running `make pep8` command in the DebOps monorepo.

**Comment by stefangweichinger** [comment]
Yeah, I see. You're strict at the line length also for ported scripts?
I wonder if we can "fork" the script and fix that in our copy, I wonder if upstream accepts shortened comments etc

**Comment by stefangweichinger** [comment]
btw: I am trying to use [black](https://black.readthedocs.io/en/stable/integrations/github_actions.html) for reformatting and that tool could also be used within a github action. Maybe interesting to you:

https://black.readthedocs.io/en/stable/integrations/github_actions.html

**Comment by stefangweichinger** [comment]
https://github.com/andreasscherbaum/ansible-lxc-ssh/pull/35 was merged right now.

https://github.com/andreasscherbaum/ansible-lxc-ssh/commit/6ce02212f6d2acdcd9f4eaf5460fa1a6bb873c1a

Committing the latest script in a minute.

**Review by drybjed — APPROVED** [review]
Thanks for working with upstream and updating the plugin. I'll backport the changes to the stable releases as well, to keep them synchronized.

**Comment by stefangweichinger** [comment]
> Thanks for working with upstream and updating the plugin. I'll backport the changes to the stable releases as well, to keep them synchronized.

Glad I could help.

**Pull request merged**